kpops/components/streams_bootstrap/producer/model.py

```diff
@@ -15,8 +16,48 @@ class ProducerAppValues(StreamsBootstrapValues):
     """Settings specific to producers.
 
     :param kafka: Kafka Streams settings
+    :param deployment: Deploy the producer as a Kubernetes Deployment (thereby ignoring Job-related configurations)
+    :param restart_policy: See https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy
+    :param schedule: Cron expression to denote a schedule this producer app should be run on. It will then be deployed as a CronJob instead of a Job.
+    :param suspend: Whether to suspend the execution of the cron job.
+    :param successful_jobs_history_limit: The number of successful jobs to retain.
+    :param failed_jobs_history_limit: The number of unsuccessful jobs to retain.
+    :param backoff_limit: The number of times to restart an unsuccessful job.
+    :param ttl_seconds_after_finished: See https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/#ttl-after-finished-controller
     """
 
     kafka: ProducerConfig = Field(description=describe_attr("kafka", __doc__))
 
+    deployment: bool = Field(
+        default=False, description=describe_attr("deployment", __doc__)
+    )
+
+    restart_policy: RestartPolicy = Field(
+        default=RestartPolicy.ON_FAILURE,
+        description=describe_attr("restart_policy", __doc__),
+    )
+
+    # TODO: we can do validation of the cron expression with https://github.com/kaka2507/cron-validator
```

Comment:
good idea! are you going to a add a Pydantic validator?

Reply:
Yes! It is done! I will push the changes